src/library/dlgtagfetcher.cpp

```diff
@@ -250,45 +293,63 @@ void DlgTagFetcher::fetchTagFinished(
 }
 
 void DlgTagFetcher::slotNetworkResult(
-        int httpError, const QString& app, const QString& message, int code) {
+        int httpError,
+        const QString& app,
+        const QString& message,
+        int code) {
     m_networkResult = httpError == 0 ? NetworkResult::UnknownError : NetworkResult::HttpError;
     m_data.m_pending = false;
-    QString strError = tr("HTTP Status: %1");
-    QString strCode = tr("Code: %1");
-    httpStatus->setText(strError.arg(httpError) + "\n" + strCode.arg(code) + "\n" + message);
```

Comment:
The error codes and messages are helpful, in case of digging down user issues. 
We need to find here a good balance here. 
Is there a need to short down the message? If yes, we may log a qWarning with the original message and put only the the `message` into the progress bar. 
Will this work?

Reply:
There can be 2 messages that comes to my mind that we can use it to inform users, "Can't connect" and "Can't find in the database".

Related messages can be logged to qWarning. 

What do you think?

Reply:
OK.